src/components/cartCard/index.tsx

```diff
@@ -16,21 +16,27 @@ const CartCard = ({itemData,removeFromCart,incNumber,decNumber}) =>{
 
             <Text style={styles.cardTitle}>{itemData.name}</Text>
             <Text style={styles.cardInfo}>{itemData.info}</Text>
-            <Icon name="trash" size={20} style={styles.cardIcon} onPress={()=>{removeFromCart(itemData.id)}}/>
-            <View style={styles.Price}>
-                <Icon name="rupee" size={20}/>
-                <Text style={styles.pricestyle}>{itemData.price}</Text>
-            </View>
 
             <View style={styles.quant}>
-                <Icon name="minus" size={20} style={styles.quantIcon} onPress={()=> {incNumber()}}/>
+                <View style={styles.quantUpDownCounter}>
+                <Icon name="minus" size={18} style={styles.quantIcon} onPress={()=> {incNumber()}}/>
                 <Text style={styles.quantText}>{itemData.qty}</Text>
-                <Icon name="plus" size={20} style={styles.quantIcon} onPress={()=> {decNumber()}}/>
+                <Icon name="plus" size={18} style={styles.quantIcon} onPress={()=> {decNumber()}}/>
+                </View>
                 <Text style={styles.quantText}>|</Text>
-                <Text style={styles.quantText}>43</Text>
-                <Icon name="chevron-down" size={20} style={styles.quantIcon}/>
+                <View style={styles.Price}>
+                <Icon name="rupee" size={18} style={styles.quantRupeeIcon}/>
+                <Text style={styles.pricestyle}>{itemData.price}</Text>
+                </View>
+               
             </View>
-
+            <TouchableOpacity style={{flexDirection:'row',alignItems:'center'}}
```

Comment:
touchable is old thing you can replace it with pressable if possible

Reply:
okay